src/transformers/training_args.py

```diff
@@ -539,6 +543,7 @@ class TrainingArguments:
     )
     no_cuda: bool = field(default=False, metadata={"help": "Do not use CUDA even when it is available"})
     seed: int = field(default=42, metadata={"help": "Random seed that will be set at the beginning of training."})
+    data_seed: Optional[int] = field(default=None, metadata={"help": "Random seed to be used with data samplers."})
```

Comment:
```suggestion
    data_seed: int = field(default=None, metadata={"help": "Random seed to be used with data samplers."})
```
Here we don't need the real type hint as it won't be seen in the doc.